persistent-mongoDB/Database/Persist/MongoDB.hs

```diff
@@ -131,7 +131,9 @@ import Data.Text (Text)
 import qualified Data.ByteString as BS
 import qualified Data.Text.Encoding as E
 import qualified Data.Serialize as Serialize
-import Web.PathPieces (PathPiece (..))
+import Web.PathPieces (PathPiece(..))
+import Web.HttpApiData (ToHttpApiData(..), FromHttpApiData(..))
+import Web.HttpApiData.Internal (parseUrlPieceWithPrefix, readEitherTextData, parseUrlPieceMaybe)
```

Comment:
Should this module be `Internal` if it is used by all the persistent libraries? Internal generally means something that should be avoided if possible and not depended upon by a library. A more usual name would be `Utils`, but I don't like generic names like that and tend to look for something more like `Web.HttpApiData.Parse`